Coolbooru/Coolbooru.cs

```diff
@@ -82,39 +193,78 @@ public class CoolImages {
 	/// Represents arguments to a search query.
 	/// </summary>
 	public class CoolSearchQuery {
-		private string pq;
-		private string psf;
+		private string pQuery = "*";
+		private string pSortFormat;
 
-		public string q {
-			get { return pq; }
-			set { pq = Uri.EscapeUriString(value); }
+		/// <summary>
+		/// Your Derpibooru API key.
+		/// </summary>
+		public string APIKey;
+		/// <summary>
+		/// The Derpibooru query to search for. Defaults to *, for all tags.
+		/// </summary>
+		public string Query {
```

Comment:
Shouldn't that be implied? It's a wrapper for a reason; though it's lower-level, we're still making a layer of abstraction on top of things.